shared/oae/api/oae.api.util.js

```diff
@@ -1656,12 +1656,15 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
                 return '';
             } else {
 
+                input = input.replace(/\*/g, "\\*");
+
                 // Convert the Markdown input string to HTML using marked.js. `gfm`
                 // automatically recognizes text beginning with http: or https: as a URL
                 // and converts it to a link. We also specify that the input should be sanitized.
                 // @see https://github.com/chjj/marked
                 input = markdown(input.toString(), {
                     'gfm': true,
+                    'tables': true,
```

Comment:
@SamPeck : Can you provide a design for how you want tables inside of a comment to be rendered (margin, padding, border style, etc.)?

<img width="1580" alt="screen shot 2016-05-27 at 12 10 39" src="https://cloud.githubusercontent.com/assets/109850/15618676/440ecfa0-2404-11e6-924d-968647f76874.png">